fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -4734,6 +4735,13 @@ public void alterTable(AlterTableStmt stmt) throws DdlException, UserException {
         this.alter.processAlterTable(stmt);
     }
 
+    /**
+     * used for handling AlterViewStmt (the ALTER VIEW command).
+     */
+    public void alterView(AlterViewStmt stmt) throws DdlException, UserException {
```

Comment:
This is the entry of ALTER VIEW operation. And the main logic is implemented in `alter` instance. But in `alter`, you call the `modifyViewDef()` in `catalog` again, which is confused.
I think `modifyViewDef()` can be implemented in `alter`.